src/main/resources/application.yml

```diff
@@ -48,6 +48,10 @@ nakadi:
       replicaFactor: 1
       retentionMs: 172800000 # 2 days
       rotationMs: 86400000 # 1 day
+    compacted:
+      rotationMs: 3600000 # 1 hour
```

Comment:
I would set Kafka defaults at the project level by now. If we get some feedback we could change it to some other value.

Reply:
Ok, makes sense. Will change it.
What about rotationMs ? Kafka default is 7 days, but in Nakadi we have 1 day. What should we define for compacted topics? 1 day?

Reply:
I mean, maybe use what we already used, 1 day. @v-stepanov let's make the min lag 3 hours as well? Like we decided in the previous alignment meeting?

Reply:
yes, sorry, I mean I changed to 1 day (not hour) :)

And I have set min lag to 3 hours now as you asked. I think it's a good thing as this is the easiest way to fix archiver problem.

Reply:
But also I think if we decided to set min lag to 3 hours - it can make sense to set segment.ms <= 3 hours. So that it can always compact after 3 hours. Because I think it will be difficult to explain users this 1day/3hour/1GB thing :) If we remove 1day it will be a bit easier as there will be only 3hour/1GB thing.